harness/determined/cli/user.py

```diff
@@ -218,6 +243,110 @@ def edit(args: argparse.Namespace) -> None:
         raise errors.CliError("No field provided. Use 'det user edit -h' for usage.")
 
 
+def render_token_info(token_info: Sequence[bindings.v1TokenInfo]) -> None:
+    values = []
+    for ti in token_info:
+        value = [
+            ti.id,
+            ti.userId,
+            ti.description,
+            ti.createdAt,
+            ti.expiry,
+            ti.revoked,
+            ti.tokenType,
+        ]
+        values.append(value)
+    render.tabulate_or_csv(TOKEN_HEADERS, values, False)
+
+
+def print_token_info(data: Sequence[bindings.v1TokenInfo], args: argparse.Namespace) -> None:
+    if data:
+        json_data = [elem.to_json() for elem in data]
+        if len(data) == 1:
+            json_data = json_data[0]
+    else:
+        json_data = {}
+    if args.yaml:
+        print(util.yaml_safe_dump(json_data, default_flow_style=False))
+    else:
+        render.print_json(json_data)
+
+
+def describe_token(args: argparse.Namespace) -> None:
+    sess = cli.setup_session(args)
+    result = []
+    for arg in args.username:
+        userID = bindings.get_GetUserByUsername(session=sess, username=arg).user.id
+        resp = bindings.get_GetAccessToken(session=sess, userId=userID)
```

Comment:
random question: what is the difference between GetAcessToken and GetAllAccessTokens?

Why would `det token describe my_user` show one token but `det token list` show multiple tokens?  How do you pick the one token if you only show one?

Reply:
`GetAcessToken` - Intends to return given user's token info. A non-admin will only be able to view their own token info, whereas admin can view any user's token info. Command run by Non-Admin : `det user token describe <their_own_username>`. Command run by Admin : `det user token describe <username1> <username2> ..` intends to return for the given usernames. 

`GetAllAccessTokens` API  - Intends to return all (active/revoked) token info. This is only accessible by admins. `det user token list -(a)/ll` CLI calls this API

Reply:
Resolving it as it is now implemented 
1. We only have GetAllAccessToken now, supporting describe (mapped per token info) and list | ls (for multiple users)